distributed/client.py

```diff
@@ -1053,6 +1053,23 @@ def _update_scheduler_info(self):
         except EnvironmentError:
             logger.debug("Not able to query scheduler for identity")
 
+    @gen.coroutine
+    def _wait_for_workers(self, func, n):
+        while n and (yield func()) < n:
+            yield gen.sleep(0.1)
+
+    def wait_for_workers(self, n_workers=0):
+        """
+        Note: If you add more arguments, you will need to return a list of
+        futures, or assert that only 1 is called at a time.
```

Comment:
This looks like it is a comment designed for other developers.  For this I recommend that you use comments (with the `#` character).  For the docstring lets follow this approach: https://docs.dask.org/en/latest/develop.html#docstrings